.github/workflows/ci-test.yml

```diff
@@ -51,14 +51,15 @@ jobs:
     - name: Install Linux Mesa OpenGL drivers and Microsoft fonts
       if: matrix.os == 'ubuntu-latest'
       run: |
-        echo "ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true" | sudo debconf-set-selections
+        echo 'ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true' | sudo debconf-set-selections
         sudo apt-get install -y mesa-utils xvfb libfontconfig1 ttf-mscorefonts-installer
         python -m pip install pytest-xvfb
     - name: Install Windows Mesa OpenGL drivers
       if: matrix.os == 'windows-latest'
       shell: cmd
       run: |
-        curl -L --output mesa.7z --url https://github.com/pal1000/mesa-dist-win/releases/download/24.3.4/mesa3d-24.3.4-release-msvc.7z
+        curl -L --output mesa.7z --url \
+          https://github.com/pal1000/mesa-dist-win/releases/download/24.3.4/mesa3d-24.3.4-release-msvc.7z
```

Comment:
Looks like Windows is absolutely not happy for this line to be split with a backslash. Not sure what (if anything) is the correct syntax for continuation in the windows command shell.

Reply:
Yeah I'm not that fluent with Windows to be honest. Glad you tracked it down!

Reply:
Yeah I'm not that fluent with Windows to be honest. Glad you tracked it down!